### ISSUE TITLE
Grimshot should pause the screen when taking screenshots.

### ISSUE THREAD

**Issue by glyh** [swaywm/sway#6899]
### Please fill out the following:
- **Description:**
Please describe in plain English what the enhancement is and what the use case is.

For example, I'd like to take a screenshot to part of the screen. I fire up grimshot to choose the area I'd like. However, during this period, the background of the screen is still moving. This would be annoying if I'm trying to take a screenshot from a video.

**Comment by Nefsen402** [comment]
This would need to be a feature of the screenshot tool, not of sway itself.

**Comment by glyh** [comment]
Isn't grimshot managed in sway?

**Comment by Nefsen402** [comment]
Ah right. Well, in order to fix this you'll need to take a screenshot of all the different outputs, display those screenshots covering up everything else so that it appears nothing is moving, then grimshot would work like normal.

**Comment by emersion** [comment]
I don't think this is in scope for grimshot, sorry.

**Comment by Absobel** [comment]
hyprpicker -r -z works fine on sway, wouldn't it be possible to freeze the screen using this, take the screen and then kill it, like they did with grimblast ?

**Comment by martinstark** [comment]
This is now achievable with grim/grimshot in Sway using `wayfreeze`, without needing to switch wm to hyprland (for grimblast), or use flameshot:

https://github.com/Jappie3/wayfreeze

`bindsym $mod+Shift+s exec wayfreeze & PID=$!; sleep .1; grimshot copy area; kill $PID`

or

`wayfreeze --after-freeze-cmd 'grim -g "$(slurp)" - | wl-copy; killall wayfreeze'`
